mailscanner/functions.php

```diff
@@ -457,37 +457,39 @@ function printServiceStatus()
         }
         echo '     <tr><td>' . __('mailscanner03') . '</td><td align="center">' . $running . '</td><td align="right">' . $procs . '</td></tr>' . "\n";
 
-        $output = 0;
         // is MTA running
         $mta = get_conf_var('mta');
-        if ('msmail' === $mta) {
-            exec('ps ax | grep postfix | grep -v grep | grep -v php', $output);
-            if (count($output) > 0) {
+        if (('msmail' === $mta) || ('postfix' === $mta)) {
+            $masterproc = [];
+            exec('ps ax | grep postfix.*master | grep -v grep',$masterproc);
```

Comment:
Please run `vendor/bin/php-cs-fixer fix` to fix code style

Reply:
OK, will do that.

btw: why is php-cs-fixer always failing here on github PRs?

Reply:
> btw: why is php-cs-fixer always failing here on github PRs?

There was a problem with GitHub workflow, it should be fixed on `1.2` branch, try rebasing your PR on that branch.